.env.sample

```
@@ -0,0 +1,8 @@
+DATABASE_URL="mysql://USERNAME:PASSWORD@127.0.0.1:3306/seoulminer"
+
+DB_CONNECTION = mysql
+DB_HOST = 127.0.0.1
+DB_USERNAME = USERNAME
+DB_PASSWORD = PASSWORD
+DB_DATABASE = seoulminer
+DB_PORT = 3306
```

- 작성해주신 파일 내용의 컨벤션이 일치하지 않고, 실제 디비명을 썼을 법한 내용이 샘플파일에 함께 올라와 있네요!

```suggestion
DATABASE_URL="dbType://userName:password@iphost:port/dbName"

DB_CONNECTION=DB_TYPE
DB_HOST=HOST_IP
DB_USERNAME=USERNAME
DB_PASSWORD=PASSWORD
DB_DATABASE=DB_NAME
DB_PORT=PORT_NUMBER
```